### DISCUSSION TITLE
overhaul for ui

### DISCUSSION THREAD

**Discussion by Amin456789** [LostRuins/koboldcpp#481] [Ideas]
Greetings, i'm back with another idea which are inspired by tavernai's features

honestly after the latest update [thank u very much] and using chat mode here and having the option to put avatar in koboldcpp, i'd rather to only use koboldcpp than to use tavernai and the others alongside it. even though i love them but with a basic change in koboldcpp's ui we can have everything in one package:

1. changes in the ui= having some tweaks such as transparency and different colors + adding a background would be really good for atmospheric adventure/story and chats. a chat panel to choose characters will be great too, which leads to the second idea:

2. storing the process and avatars/background images in the same folder as koboldcpp= let's say we had a chat and uploaded an avatar, it will be great just to run the gui and see what we left and continue, for making it happen we can have an command argument such as -chats ./chat folder which will make/save/load the new chats in that folder and our process in our koboldcpp.exe folder, same can be used for adventure and story
it can be optional for whoever wants it or not

kind regards

**Comment by LostRuins** [comment]
kobold lite is designed to run from browser alone - which means that all resources must be handled within the webpage. There are actually a bunch of new features about to come in the next release such as Local Storage Saves and Group Chat Character Toggles, you can check it out early on web https://lite.koboldai.net

**Comment by Amin456789** [comment]
thank u so much! can't wait for the upcoming features!

**Reply by LostRuins** [reply]
next release is out!

**Reply by Amin456789** [reply]
thnx, here we gooo!
